packages/integration-tests/projects/suite-web/tests/wallet/add-account-types.test.ts

```diff
@@ -69,4 +73,110 @@ describe('Add-account-types', () => {
                 }),
         );
     });
+
+    /**
+     * Test case
+     * 1. go to Settings
+     * 2. activate LTC
+     * 3. go to Accounts
+     * 4. Unpack all account types
+     * 5. Get the number of accounts
+     * 6. Create new account for each account type
+     * 7. Get the number of accounts again
+     * 8. Verify that the current number is equeal to previous number + 1
+     */
+    it('Add-account-types-LTC', () => {
+        //
+        // Test preparation
+        //
+        const coin = 'ltc';
+        const accsArray = [
+            ['normal', 'Litecoin'],
```

Comment:
I'd personally make this nested data structure object instead of array
const accsArray = [{type: 'normal', coin: 'Litecoin'}]